erikbenedict/fruit-preserves
# chore: added screenshot to images file; updated README.md

## Test

- navigate to fruit-preserves repository
- scroll down to fins the README
- proof read, test deployed linked, see that a screenshot is visible
